tests/integration/test_charm.py

```diff
@@ -315,3 +316,17 @@ async def assert_links_in_configmap_by_text_value(
         assert item.text in links_texts
 
     return links_texts
+
+
+@pytest.mark.asyncio
+async def test_dashboard_access(ops_test: OpsTest, lightkube_client: Client):
+    namespace = ops_test.model_name
+    application_ip = lightkube_client.get(Service, CHARM_NAME, namespace=namespace).spec.clusterIP
+    config = await ops_test.model.applications[CHARM_NAME].get_config()
+    application_port = config["port"]["value"]
+    url = "http://" + str(application_ip) + ":" + str(application_port)
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get(url, headers=None) as response:
+            result_status = response.status
+    assert result_status == 200
```

Comment:
suggestion: add another assert that `Kubeflow Central Dashboard` is in the response text

Reply:
Added an assertion to test the contents of <title>